pkg/storage/ibmcos/powervs_regions.go

```diff
@@ -0,0 +1,61 @@
+package ibmcos
+
+import (
+	"fmt"
+)
+
+// powerVSRegion describes respective IBM Cloud region and PowerVS zones associated with a region in PowerVS.
+// which helps to decide COS location from PowerVS region with the intention to keep COS in same region as PowerVS.
```

Comment:
The mix of terms here is confusing. You're using:
* IBM cloud region
* PowerVS zone
* PowerVS region
* COS location

Could you rephrase this comment so that it's easier to understand for someone who isn't familiar with ibm cloud?

Reply:
Tried rephrase the comment to make it clear on why this is required, ptal!

Reply:
Rephrased comment is much more clear - thank you!